lib/telegraf/telegraf.conf.erb

```diff
@@ -73,7 +73,7 @@
   prefix = ""
   ## Stats output template (Graphite formatting)
   ## see https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_OUTPUT.md#graphite
-  template = "host.tags.measurement.field"
+  template = "host.measurement.tags.field"
```

Comment:
Once this is merged, we need to re-run everything and make sure we're doing for all other metrics.